OneCommunityGlobal/HighestGoodNetworkApp
# Anirudh taking over for Swathy - Unit test TeamsAndProjects/UserTeamsTable.jsx

# Description
Unit testing of User Teams Table component under User Profile.

## Main changes explained:
- Create a unit test file for User Teams Table Component under User Profile --> Teams and Projects --> ```__tests__```

## How to test:
1. check into current branch
2. run ```"npm test src/components/UserProfile/TeamsAndProjects/__tests__/UserTeamsTable.test.js"```

## Screenshots or videos of changes:
You should be able to see 6 successful tests
<img width="1285" alt="Screenshot 2024-04-05 at 6 48 07 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/13806470/d7a2e4f3-1b24-41c2-80c3-e95d9af35c6e">

## Note:
Include the information the reviewers need to know.
